docs/changes/16181.other.rst

```diff
@@ -0,0 +1,3 @@
+Change the default value of ``copy`` arguments in public APIs from ``False`` to
+``None`` if Numpy 2.0 or newer is installed.
+For details, see :ref:`What's New in Astropy 6.1?<_whatsnew-6.1-copy-semantics>`
```

Comment:
This failed RTD build with a warning:

```
WARNING: undefined label: '_whatsnew-6.1-copy-semantics'
```

Reply:
Also I don't think intersphinx is going to last forever in What's New as we "archive" the page at release time. For example, please see https://github.com/astropy/astropy/blob/main/docs/whatsnew/6.0.rst

Reply:
Thanks for pointing it out, how can I fix this ? Should I just refer to the what's new page in text and avoid any linking ?

Reply:
I am okay with a mention without any link. What do you think, @mhvk ?

Reply:
Agreed, let's not over{th|l}ink.